### PR TITLE
WT-6669 Enable VLCS coverage and checkpoint tests in evergreen

### PR DESCRIPTION
**Pull request by keithbostic**

### CODE REVIEW

**Comment by keithbostic** [comment]
@lukech, the coverage tests look like they're pretty much the same as we had in Jenkins. I'm not sure what the coverage tests are actually used for, but the configurations we're using are likely out of date (for example, we explicitly test zlib compression but don't test Zstd compression which is the standard now, I believe). Anyway, pointing that out in case there's reason to revisit the current coverage configurations.

File: test/evergreen.yml [hunk 1]
```diff
@@ -2104,14 +2104,12 @@ tasks:
         vars:
           # Don't use diagnostic - this test looks for timing problems that are more likely to occur without it
           posix_configure_flags: --enable-strict
-      # Temporarily disable mix and column file type
-      # FIXME after WT-6608
```
Comment (lukech) [inline test/evergreen.yml:2108]: I guess the issue reported in WT-6608 had been addressed by WT-7928?

**Comment by lukech** [comment]
@keithbostic The `coverage-report` Evergreen task runs a series of tests and generates a code coverage report (one [example](https://build_external.s3.amazonaws.com/wiredtiger/code-statistics/026a930bb418933cddd54d213a39ada253cde1d0/coverage_report_wiredtiger_code_statistics_026a930bb418933cddd54d213a39ada253cde1d0_21_08_30_00_52_16-0/1_coverage_report_main.html)) to help us evaluate/improve testing coverage. It's a good catch `zstd` is not part of the task which it's supposed to be. WT-8037 was filed to review the task to see what we could improve in terms of testing coverage.

**Review by lukech — APPROVED** [review]
LGTM. Suggest creating a patch build to run the `checkpoint-filetypes-test` and `coverage-report` tasks before merging.

File: test/format/CONFIG.coverage [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# A reasonable configuration for coverage testing.
+btree.huffman_value=0
+cache.minimum=20
+runs.rows=1000000:5000000
+runs.threads=4:32
+runs.timer=15
+runs=100
```
Comment (keithbostic) [inline test/format/CONFIG.coverage:7]: I'd remove `runs=100` (we don't want 100 runs, and it's being  overridden by the command line anyway).

File: test/evergreen.yml [hunk 3]
```diff
@@ -2129,36 +2131,44 @@ tasks:
...
         vars:
+          config: ../../../test/format/CONFIG.coverage
           extra_args: leak_memory=0 mmap=1 file_type=row checkpoints=0 in_memory=1 reverse=1 truncate=1
       - func: "format test"
         vars:
+          config: ../../../test/format/CONFIG.coverage
           extra_args: checkpoints=1 leak_memory=0 mmap=1 file_type=row compression=zlib huffman_value=1
       - func: "format test"
         vars:
+          config: ../../../test/format/CONFIG.coverage
```
Comment (keithbostic) [inline test/evergreen.yml:2154]: I think `leak_memory=0 mmap=1` are common to all runs, so they could go move to `CONFIG.coverage`.

There's one test that doesn't have `checkpoints=1`, but I think that's likely not to be useful, so I'd move `checkpoints=1` into `CONFIG.coverage` as well.

Comment (keithbostic) [inline test/evergreen.yml:2154]: I think `leak_memory=0 mmap=1` are common to all runs, so they could go in `CONFIG.coverage`.

Comment (keithbostic) [reply test/evergreen.yml:2154]: @lukech, that's correct, additional code paths but not replacement code paths.

**Pull request merged**